check_whitespace.c

```diff
@@ -67,6 +67,10 @@ int is_clean(char* str) {
   // greater than the second.
   result = strcmp(str, cleaned);
 
+  if (cleaned[0] != (char) 0) {
```

Comment:
What you have is correct, but you would typically write this as:

```
if (cleaned[0] != `\0') {
```

Alternatively you could use `strcmp` to see if `cleaned` is the empty string.